src/ciphersuite/codec.rs

```diff
@@ -48,6 +48,13 @@ impl Codec for Signature {
     }
 }
 
+impl tls_codec::TlsSize for Signature {
+    #[inline]
+    fn serialized_len(&self) -> usize {
+        2 + self.value.len()
```

Comment:
It would be good to have a comment when const values are used, so that we know what to change in case the const changes some day.

Reply:
Not sure I understand.

Reply:
I think what @raphaelrobert means is that we have a `2` here without an explanation of why it's `2` and not some other constant. (And the same for the comments below.)